### PR TITLE
Feature/contact

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**

### CODE REVIEW

File: practice-two/src/assets/styles/layouts/_contact.scss [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        }
+      }
+      .btn-secondary {
+        padding: 12px 23px;
+      }
+    }
+    .form-text {
+      font-size: 12px;
+      line-height: 14px;
+      color: $color-green;
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/assets/styles/layouts/_contact.scss:78]: @daoanhhuy26012001 `$color-green` can not be reusable.
ex: The design will use `blue` color, how to update it?

File: practice-two/src/index.html [hunk 2]
```diff
@@ -126,6 +126,42 @@ <h2 class="card-title">
...
       </section>
+      <section class="contacts">
+        <div class="contacts-container">
+          <div class="members-item">
+            <p class="members-name">Frankie</p>
+            <p class="members-describe">Member since 2016</p>
+          </div>
+          <div class="members-item">
+            <p class="members-name">Camile</p>
+            <p class="members-describe">Member since 2012</p>
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/index.html:137]: @daoanhhuy26012001 `describe` is a verb.  In this case, please use a noun. `des` stand for description is enough.

File: practice-two/src/index.html [hunk 3]
```diff
@@ -148,20 +148,20 @@ <h2 class="contacts-title">
...
               id="form-control"
               placeholder="Enter your email"
             />
-            <p class="form-text form-text-message">Thanks! Email received.</p>
+            <p class="form-message form-text-message">Thanks! Email received.</p>
             <button class="btn btn-secondary">join our list</button>
           </form>
           <div class="notification">
-            <p class="form-text">Thanks! Email received.</p>
+            <p class="form-message">Thanks! Email received.</p>
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/index.html:161]: @daoanhhuy26012001 Line 157 is the same as line 161. Is it right?

Comment (daoanhhuy26012001) [reply practice-two/src/index.html:161]: Yes, It is

**Pull request merged**